core/jazz_create-serverless-service/index.js

```diff
@@ -234,8 +234,8 @@ var getServiceData = (service_creation_data, authToken, configData) => {
     let primaryAccountCount = 0;
     let deployment_accounts = [];
     // TODO: Make this field required when we support multiple accounts/regions/providers
```

Comment:
Remove this comment as deployment_accounts field is included in required filed.

Reply:
Removed the comment